Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -101,6 +101,10 @@ let Keyboard = {
    *
    * Note that if you set `android:windowSoftInputMode` to `adjustResize`  or `adjustNothing`,
    * only `keyboardDidShow` and `keyboardDidHide` events will available on Android.
+   * 
+   * Limitation for keyboardWillShow: On Android no event is fired when the keyboard is about
+   * to be opened. Instead you can use a layout listener to check if the height of the layout
```

Comment:
[This StackOverflow page](https://stackoverflow.com/questions/25216749/softkeyboard-open-and-close-listener-in-an-activity-in-android)  and several others suggest that there is indeed no way on Android to do this.